CurrentLocationKotlin/README.md

```diff
@@ -0,0 +1,48 @@
+Current Location (Kotlin)
+===========================================
+Demonstrates retrieving the current location in Kotlin.
+
+Introduction
+============
+This app allows a user to receive the current location without subscribing to location updates (and
+then unsubscribe once you retrieve the location).
+
+Users have four options in Android 11+ regarding location:
+* One time only
+* Allow while app is in use, i.e., while app is in foreground
+* Allow all the time
+* Not allow location at all
+
+In this samples case, we only ask for 'while-in-use' location access via the FINE location permission
+(`android.permission.ACCESS_FINE_LOCATION`), so the **All the time** access will not show up as an
```

Comment:
should we add links to DAC for `ACCESS_FINE_LOCATION` and other places?

Reply:
Added link and another link to a guide.